js_modules/dagster-ui/package.json

```diff
@@ -18,9 +19,12 @@
   },
   "packageManager": "yarn@3.6.4",
   "dependencies": {
-    "graphql.macro": "^1.4.2"
+    "graphql.macro": "^1.4.2",
+    "patch-package": "^8.0.0",
+    "postinstall-postinstall": "^2.1.0"
   },
   "devDependencies": {
+    "@types/dagre": "^0",
```

Comment:
Does this one need to be here? It feels odd because the patcher doesn't need to be able to parse the src?